bittensor/_neuron/text/core_validator/__init__.py

```diff
@@ -1232,6 +1233,11 @@ def _synergy(first, second, target, ext):
     logger.info(f'{str(synapse)} \t| Shapley synergy values <dim>[{time.time() - synergy_start_time:.3g}s]</dim>')
 
     if logging:
+        # === Response table ===
+        # Prints the query response table: top prediction probabilities and texts for batch tasks
+        batch_predictions = format_predictions(uids, query_responses, return_ops, inputs, validation_len, index_s)
+        response_table(batch_predictions, stats, sort_col='shapley_values_nxt', console_width=console_width)
```

Comment:
Would this be too much for the average user? Would it be beneficial to put another level of debugging?

Reply:
Potentially yes, we could add a flag to make this optional later. Right now the log space is dominated by IPFS dataset loaded and receptor create/destroy lines which far exceed space taken up by these neuron stat overviews, so I'd say this PR adds a relatively acceptable amount of log bloat.